packages/flutter_tools/lib/src/ios/ios_emulators.dart

```diff
@@ -37,7 +37,7 @@ class IOSEmulator extends Emulator {
 
   @override
   Future<void> launch() async {
-    Future<void> launchSimulator(List<String> additionalArgs) async {
+    Future<bool> launchSimulator(List<String> additionalArgs) async {
```

Comment:
The ambient `launch()` also returns `Future<bool>`

Reply:
It doesn't seem to be used as `Future<bool>` that I can see - the result of `awaits` on it are ignored?  I'll change the return statement to reflect this, but let me know if the correct fix is to change the return type.

Reply:
Seems to be the result of a simple refactoring error. @DanTup would know. Given that the base class uses `Future<void>`, I think we can move on with the change of return type.

Reply:
I think these should all be Future<void> (see 48f4ff6ddab5520a6a761f4e6079c64000c16d29). IIRC we used to return a bool of whether the launch worked and just print the error, but this didn't work when we exposed it in the daemon; so they all became Future<void> and we throw on failure.

Reply:
So I think the line `if (!await launchSimulator(<String>['-n']))` no longer needs any condition/return, because it'll throw if it fails. I must've missed that when I changed from bool -> void.

Reply:
@leafpetersen Not sure of the status of this - let me know if you want me to fix it up (I believe everything should be `void`, no `bool`s).

Reply:
If you don't mind, how about you fix it up as you see best?  I'm not sure I understood exactly what changes were implied by your second to last comment here.  Thanks!

Reply:
See comment below - I got confused between the two methods - I think your changes are exactly correct.